modules/formulize/libraries/formulize-admin.js

```diff
@@ -15,19 +15,28 @@ jQuery(document).ready(function() {
         }
         hidden_section.toggle();
     });
-    
+
 });
 
-function fz_check_php_code(custom_code, block_name, icms_url, icms_userid) {
+async function fz_check_php_code(custom_code, block_name) {
```

Comment:
Since this function itself does not make use of the `await` keyword there's no need to call it an async function. 
```suggestion
function fz_check_php_code(custom_code, block_name) {
```

Reply:
Confirmed. Thanks!